[2025-10-18T14:26:14.767+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-10-18T14:26:14.778+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: Anomaly_Detection_Daily.calculate_forecast_range manual__2025-10-18T14:26:04.789931+00:00 [queued]>
[2025-10-18T14:26:14.786+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: Anomaly_Detection_Daily.calculate_forecast_range manual__2025-10-18T14:26:04.789931+00:00 [queued]>
[2025-10-18T14:26:14.787+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 1
[2025-10-18T14:26:14.807+0000] {taskinstance.py:2888} INFO - Executing <Task(PythonOperator): calculate_forecast_range> on 2025-10-18 14:26:04.789931+00:00
[2025-10-18T14:26:14.816+0000] {logging_mixin.py:190} WARNING - /home/airflow/.local/lib/python3.12/site-packages/airflow/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=1171) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-10-18T14:26:14.818+0000] {standard_task_runner.py:72} INFO - Started process 1187 to run task
[2025-10-18T14:26:14.816+0000] {standard_task_runner.py:104} INFO - Running: ['airflow', 'tasks', 'run', 'Anomaly_Detection_Daily', 'calculate_forecast_range', 'manual__2025-10-18T14:26:04.789931+00:00', '--job-id', '72', '--raw', '--subdir', 'DAGS_FOLDER/anomaly_detection.py', '--cfg-path', '/tmp/tmpkpeexqll']
[2025-10-18T14:26:14.818+0000] {standard_task_runner.py:105} INFO - Job 72: Subtask calculate_forecast_range
[2025-10-18T14:26:14.846+0000] {logging_mixin.py:190} WARNING - /home/airflow/.local/lib/python3.12/site-packages/airflow/settings.py:209 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-10-18T14:26:14.896+0000] {task_command.py:467} INFO - Running <TaskInstance: Anomaly_Detection_Daily.calculate_forecast_range manual__2025-10-18T14:26:04.789931+00:00 [running]> on host a195a1fbbb6f
[2025-10-18T14:26:14.987+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='Anomaly_Detection_Daily' AIRFLOW_CTX_TASK_ID='calculate_forecast_range' AIRFLOW_CTX_EXECUTION_DATE='2025-10-18T14:26:04.789931+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-10-18T14:26:04.789931+00:00'
[2025-10-18T14:26:14.988+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-10-18T14:27:08.749+0000] {logging_mixin.py:190} WARNING - /home/airflow/.local/lib/python3.12/site-packages/statsmodels/base/model.py:607 ConvergenceWarning: Maximum Likelihood optimization failed to converge. Check mle_retvals
[2025-10-18T14:27:38.124+0000] {logging_mixin.py:190} WARNING - /home/airflow/.local/lib/python3.12/site-packages/statsmodels/base/model.py:607 ConvergenceWarning: Maximum Likelihood optimization failed to converge. Check mle_retvals
[2025-10-18T14:27:38.700+0000] {logging_mixin.py:190} WARNING - /home/airflow/.local/lib/python3.12/site-packages/statsmodels/base/model.py:607 ConvergenceWarning: Maximum Likelihood optimization failed to converge. Check mle_retvals
[2025-10-18T14:28:02.799+0000] {logging_mixin.py:190} WARNING - /home/airflow/.local/lib/python3.12/site-packages/airflow/utils/email.py:155 RemovedInAirflow3Warning: Fetching SMTP credentials from configuration variables will be deprecated in a future release. Please set credentials using a connection instead.
[2025-10-18T14:28:02.800+0000] {email.py:272} INFO - Email alerting: attempt 1
[2025-10-18T14:28:02.996+0000] {email.py:283} INFO - Sent an alert email to ['subhrosmukherjee@gmail.com']
[2025-10-18T14:28:03.720+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-10-18T14:28:03.733+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-10-18T14:28:03.733+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=Anomaly_Detection_Daily, task_id=calculate_forecast_range, run_id=manual__2025-10-18T14:26:04.789931+00:00, execution_date=20251018T142604, start_date=20251018T142614, end_date=20251018T142803
[2025-10-18T14:28:03.787+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-10-18T14:28:03.800+0000] {taskinstance.py:3900} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-10-18T14:28:03.801+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
